tests/modules/entity_version_test/entity_version_test.info.yml

```diff
@@ -1,10 +1,10 @@
-name: Entity Version History Test
-description: Test module to test the entity version history functionalities.
+name: Entity Version Test
```

Comment:
Why moving this test module? It was used only in the entity_version_history module testing the History tab, overview and other functionalities related to the entity_version_history module. I think it should stay in the submodule.